core/src/main/scala/org/apache/spark/api/python/StreamingPythonRunner.scala

```diff
@@ -68,17 +68,11 @@ private[spark] class StreamingPythonRunner(
     envVars.put("SPARK_BUFFER_SIZE", bufferSize.toString)
     envVars.put("SPARK_CONNECT_LOCAL_URL", connectUrl)
 
-    val prevConf = conf.get(PYTHON_USE_DAEMON)
-    conf.set(PYTHON_USE_DAEMON, false)
```

Comment:
cc @WweiL

Reply:
Thanks for the ping! Let me close this PR then https://github.com/apache/spark/pull/45448